src/xdpd/management/plugins/rest/xcli

```diff
@@ -302,6 +339,29 @@ def execute(socket, command, req_confirm=False):
 		elif "destroy" in command:
 			data = ""
 			return rest(socket, compose_url(stripped), data, "DELETE")
+
+		elif "add" in command:
+			data = ""
+			if len(stripped) < 5:
+				return "Usage: exec add controller <lsi_name> <mode:ip:port>"
+			if "controller" in command:
+				data = parse_add_controller(stripped[4:])
+				try: 
+					return rest(socket, compose_url(stripped[:4]), data, "PUT")
+				except Exception as e:
+					return "error: "+str(e)
+			else:
+				return "Usage: exec add controller <lsi_name> <mode:ip:port>"
+
+		elif "remove" in command:
```

Comment:
This entire block, until line 364 is indented with spaces. Please move it to hard tabs.

(We don't care about PEP8, tabs were created to indent!)